services/server/config/config.go

```diff
@@ -138,6 +138,7 @@ type Debug struct {
 	UID     int    `toml:"uid"`
 	GID     int    `toml:"gid"`
 	Level   string `toml:"level"`
+	Format  string `toml:"format"`
 }
```

Comment:
Please add a comment line to explain that this is about log format

Reply:
Added!